Program.cs

```diff
@@ -717,26 +715,6 @@ private static void pauseIfInteractive()
                 Console.WriteLine();
         }
 
-        private static bool updateAppveyorVersion(string version)

```

Comment:
See https://github.com/ppy/osu-deploy/pull/160/commits/0c96adb0e9b3dfdfd969c00faeaf0804fe45a2de for explanation why.